lib/manifest.js

```diff
@@ -50,7 +50,11 @@ BroccoliManifest.prototype.write = function(readTree, destDir) {
     });
 
     includePaths.forEach(function (file) {
-      lines.push(typeof pathPrefix === 'function' ? pathPrefix(file) : pathPrefix + file);
+      if(typeof pathPrefix === 'function'){
+        lines.push(pathPrefix(file));
+      }else{
+        lines.push(file.startsWith("http") ? file : pathPrefix + file);
```

Comment:
May want to use a regex here, in case someone has a file called `http.jpg` or something. 

Example:
```js
/^http[s]*:\/{2}/.test("https://google.com") // true
```